### DISCUSSION TITLE
Does DurationFormat handle negative durations?

### DISCUSSION THREAD

**Discussion by kartva** [unicode-org/icu4x#4976] [Duration Formatter]
The [TC39 ECMAScript standard for DurationFormat, section 1.1.14.4.f.iii.3](https://tc39.es/proposal-intl-duration-format/#sec-partitiondurationformatpattern) seems to have some logic regarding negative numbers in the duration fields. I tried tracing the formatting logic through the standards, but I'm having trouble understanding whether the sign is ignored or not.

@younies @sffc

**Comment by sffc** [comment]
Yes, it supports negative durations. The spec should work with them. 

Details: https://github.com/tc39/proposal-intl-duration-format/issues/169#issuecomment-1796495625
